optionals/variants_milgp/mmac.hpp

```diff
@@ -48,157 +48,157 @@
 
 // Light Grey
 class milgp_v_mmac_light_rgr;
-class CLASS(mmac_light_gry):milgp_v_mmac_light_rgr {
+class CCLASS(mmac_light_gry):milgp_v_mmac_light_rgr {
```

Comment:
```suggestion
class CCLASS(mmac_light_gry): milgp_v_mmac_light_rgr {
```
Missing space in each class definition after `:`.